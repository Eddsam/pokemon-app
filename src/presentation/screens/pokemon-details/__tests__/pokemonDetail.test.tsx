const TestRenderer = require("react-test-renderer");
import { PokemonDetails } from "..";

describe("PokemonDetails", () => {
  it("renders correctly in Snapshot", () => {
    const tree = TestRenderer.create(<PokemonDetails />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
