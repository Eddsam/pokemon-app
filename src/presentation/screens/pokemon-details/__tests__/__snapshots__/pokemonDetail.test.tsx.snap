// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonDetails renders correctly in Snapshot 1`] = `
<div
  className="MuiGrid2-root MuiGrid2-container MuiGrid2-direction-xs-column css-npoq9a-MuiGrid2-root"
>
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation5 MuiCard-root css-epwkos-MuiPaper-root-MuiCard-root"
    style={
      {
        "--Paper-shadow": "0px 3px 5px -1px rgba(0,0,0,0.2),0px 5px 8px 0px rgba(0,0,0,0.14),0px 1px 14px 0px rgba(0,0,0,0.12)",
      }
    }
  >
    <div
      className="MuiCardHeader-root css-149tc0y-MuiCardHeader-root"
    >
      <div
        className="MuiCardHeader-content css-1jkp9bu-MuiCardHeader-content"
      >
        <span
          className="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-16xl4zq-MuiTypography-root"
          style={{}}
        >
          Missigno
        </span>
      </div>
    </div>
    <img
      className="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-1kuwa16-MuiCardMedia-root"
    />
    <div
      className="MuiCardContent-root css-1lt5qva-MuiCardContent-root"
    >
      <div
        className="MuiBox-root css-j7qwjs"
      />
      <div
        className="MuiBox-root css-5ax1kt"
      />
    </div>
  </div>
</div>
`;
